vpr/src/base/vpr_constraints_writer.h

```diff
@@ -10,7 +10,20 @@
  * Routines related to writing out the file are in vpr/src/base/vpr_constraints_serializer.h. For more information on how
  * the writing interface works, refer to vpr/src/route/SCHEMA_GENERATOR.md
  *
+ * The option --write_vpr_constraints can be used to generate the constraints files.
  *
+ * Users can also use the VPR option --floorplan_split to specify which constraints files they would like to generate.
+ * The options for floorplan_split include halves, quadrants, sixteenths and one_spot. One constraints file will be generated
+ * in each case. The default setting for floorplan_split is halves.
+ *
+ * If the user specified halves, two floorplanning partitions would be created which each take up one half of the grid.
+ * The generated constraints file would assign all primitives to one of the two partitions that were created.
+ * The same idea applies for the quadrants and sixteenths options, except that the floorplan partitions would split the
+ * grid into quadrants and sixteenths respectively, as the names imply.
+ *
+ * If the one_spot option is specified, all blocks will be locked to their original placement spot.The vpr options
+ * --place_constraint_subtile and --place_constraint_expand can be used in this case to specify whether
+ * the blocks should also be locked down to a particular subtile, or whether the constraint region should expand beyond one spot.
  */
```

Comment:
mention that right now, these are routines that create constraint files for testing cases